README.md

```diff
@@ -11,6 +11,15 @@ Params:
 ### s3-lifecycle
 Checks if "S3 Lifecycle configuration" is set for s3 buckets
 
+Params:
+- excludeBuckets: Skip compliance check for listed buckets
+    - key: excludeBuckets
+    - value: Comma separated list of buckets, e.g "bucket1, bucket2"
+
+
+### s3-vpc-traffic-only
+Checks if there are some vpc deny set for s3 buckets
```

Comment:
"S3 VPC Traffic Only checks if there is a deny rule present in the S3 policy to block non-VPC traffic"

Reply:
Will change and the next commit

Reply:
Solved here https://github.com/in4it/aws-config-compliance-checks/pull/7/commits/5029c9409b220fa10b85c83ec15ece42fe269498